src/components/Tab/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import MuiTab from '@material-ui/core/Tab';
+import Typography from '../Typography';
```

Comment:
I'm not sure if this matters, but when I was reviewing other components

Everything is imported using `@components/ComponentName'. Unless it is a direct relation/subcomponent.

Reply:
Yup, makes sense.  I think it's a little bit of a tomāto / tomäto situation, but I'll change it to keep it consistent with the other imports in the package.